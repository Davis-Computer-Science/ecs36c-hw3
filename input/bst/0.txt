e
i 1 10
i 2 20
i 3 7
i 4 4
i 6 3
i 5 5
i 9 9
i 8 8
i 11 11
t
b
h
