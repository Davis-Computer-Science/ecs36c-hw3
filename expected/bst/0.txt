3 4 7 10 20
3 4 5 8 9 11 20
4
